SwiftRustIntegrationTestRunner/SwiftRustIntegrationTestRunner/Vec.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  String.swift
+//  SwiftRustIntegrationTestRunner
+//
+//  Created by Frankie Nwafili on 2/18/22.
+//
+
+import Foundation
+
+func send_bytes(vec _: RustVec<UInt8>) {}
```

Comment:
Lets call these `swift_arg_vec_u8` and `swift_return_vec_u8`.

This way we can more easily distinguish these from other FFI functions.